src/models.py

```diff
@@ -0,0 +1,36 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+import auth_util
+
+db = SQLAlchemy()
+migrate = Migrate()
+
+class User(db.Model):
+    user_id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(120), unique=True)
+    hashed_pw = db.Column(db.String(120))
+    email_verified = db.Column(db.Boolean())
+    email_verification_secret = db.Column(db.String(64))
+
+    def __init__(self, email, hashed_pw):
+        self.email = email
+        self.hashed_pw = hashed_pw
+        self.email_verified = False
```

Comment:
Don't think we'll need to include this here if we move the default to the actual database column.